test/Api.Tests/TestStartup.cs

```diff
@@ -14,7 +12,9 @@ public void ConfigureTestServices(IServiceCollection services)
             base.ConfigureServices(services);
 
             // Register here the dependency injection services
-           // services.AddDbContext<ApplicationDbContext>(opts => opts.UseInMemoryDatabase());
```

Comment:
I did not annoy them. I just requested them to give some suggestion for my issue for which they replied me that. I am sorry for annoying everyone. I tried my level best. Thats all I can say.